src/config/constants.ts

```diff
@@ -28,10 +29,10 @@ export const SAFE_TOKEN_ADDRESSES: { [chainId: string]: string } = {
 }
 
 // Safe Apps
-export const SAFE_APPS_POLLING_INTERVAL = process.env.NODE_ENV === 'test' ? 4500 : 15000
+export const SAFE_APPS_INFURA_TOKEN = process.env.NEXT_PUBLIC_SAFE_APPS_INFURA_TOKEN || INFURA_TOKEN
+export const SAFE_APPS_POLLING_INTERVAL = 15_000
```

Comment:
Why remove the period when polling?

Reply:
It's still there, I just moved the infura var up.

Reply:
`SAFE_APPS_POLLING_INTERVAL` was dependent on the `NODE_ENV`.

Reply:
We don't have that set anywhere, and it seems it didn't affect anything.

Reply:
It's used in `useAppIsLoading` for tests but I see that they're still working.

Reply:
I meant that NODE_ENV isn't set anywhere.

Reply:
Jest [sets it to `"test"` by default](https://jestjs.io/docs/environment-variables#node_env).

Reply:
This came from safe-react. Was renamed here (https://github.com/safe-global/safe-react/commit/918d68eb01b918f2994cb26bdc3de912d414560c). I think we can remove the process.env check without problems as there is no specific test for apps not loading.

Reply:
We can rename the constant as well as the name perhaps does not make sense

Reply:
Should I rename it to `SAFE_APPS_LOAD_TIMEOUT`?

Reply:
For me fine if you want to do it. If not we can do it in one of our PRs

Reply:
I'll leave it as is for now then.